bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/PWA.java

```diff
@@ -0,0 +1,95 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2020 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+package com.adobe.cq.wcm.core.components.models;
+
+/**
+ * Interface for reading progressive web apps configuration
+ *
+ * @since com.adobe.cq.wcm.core.components.models 12.18.0
+ */
+public interface PWA {
+
+    /**
+     * Name of the property that defines it PWA is enabled for the website.
+     *
+     * @since com.adobe.cq.wcm.core.components.models 12.18.0
+     */
+    String PN_ENABLE_PWA = "enablePWA";
```

Comment:
I'd rename this property and this field as: 
String PN_PWA_ENABLED = "pwaEnabled";